### PR TITLE
Autogitops updates to add loderunner apps to preprod

### PR DESCRIPTION
**Pull request by PurpleBriar**
# Type of PR

- [ ] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [x] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR

Added yaml files and variables to enable autogitops to deploy loderunner-api/clientmode/ui to preprod. Endpoints have only been created for loderunner-api and loderunner-ui in centralus, since the intention is to demo these apps in that cluster

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

## Issues Closed or Referenced

- References retaildevcrews/ngsa-asb/issues/102

### CODE REVIEW

File: autogitops/preprod/loderunner-pod-identity.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    - objectName: CosmosKey
+      key: key
+  - secretName: CosmosUrl
+    type: Opaque
+    data:
+    - objectName: CosmosUrl
+      key: key
+  parameters:
+    usePodIdentity: "true"
+    keyvaultName: "kv-aks-fykg5bqasutle"
```
Comment (AAkindele) [inline autogitops/preprod/loderunner-pod-identity.yaml:55]: should this be using the `keyvaultNamePreprod` template variable?

File: autogitops/preprod/loderunner-ui.yaml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    name: http
+  selector:
+    app: {{gitops.lrUiName}}
+    
+---
+
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: {{gitops.namespace}}-vs
```
Comment (AAkindele) [inline autogitops/preprod/loderunner-ui.yaml:91]: I noticed the Service is using `gitops.lrUiName`. Should this also use the same name variable? ex: `{{gitops.lrUiName}}-vs` or is using the namespace intentional?

**Review by AAkindele — APPROVED** [review]
Had a couple questions, but looks good overall.

File: autogitops/preprod/loderunner-ui.yaml (hunk 2 above)
Comment (PurpleBriar) [reply autogitops/preprod/loderunner-ui.yaml:91]: The intention is to use the namespace. It should be named loderunner-vs

File: autogitops/preprod/loderunner-pod-identity.yaml (hunk 1 above)
Comment (PurpleBriar) [reply autogitops/preprod/loderunner-pod-identity.yaml:55]: I think so too. I noticed this pattern in dev, so I'll ask around to understand why.

**Pull request merged**